src/perf/lib/PerfBase.h

```diff
@@ -0,0 +1,56 @@
+/*++
```

Comment:
Let's be consistent in file naming schemes here. I think "perf_base.h" would be more consistent.

Reply:
Or rename perfioctls.h to PerfIoctls.h